<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>711</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab{

background:#EA9D9D;

color:white;

}

QTabBar::tab:selected{

background:#76A4F4;

color:white;

}

QTabWidget::pane { /* The tab widget frame */

border-top: 0px solid #EA9D9D;

}

QTabWidget::tab-bar {

left: 5px; /* move to the right by 5px */

}

tab</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>acte medicale</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{

background:#EA9D9D;

color:white;

}

QTabBar::tab:selected{

background:#76A4F4;

color:white;

}

QTabWidget::pane { /* The tab widget frame */

border-top: 0px solid #EA9D9D;

}

QTabWidget::tab-bar {

left: 5px; /* move to the right by 5px */

}

tab</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>donnez le type :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>donnez le prix</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>190</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{

background-color:rgba(0,0,0,50%);

color:white;

border:1 solid white;

border-radius: 5px;

}</string>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>60</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>120</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {

gridline-color: white;

color:white;

background-color:rgba(0,0,0,50%);

border:0px;

border-radius: 5px;

}QTableView QTableCornerButton::section {

bottom-style:none;

border:0px;



}

</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>saisir le type:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>190</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>saisir le nouveau prix</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>230</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;modifier&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="accessibleName">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{

background-color:rgba(0,0,0,50%);

color:white;

border:1 solid white;

border-radius: 5px;

}</string>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>100</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>190</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>supprimer</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>saisir le type:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>160</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{

background-color:rgba(0,0,0,50%);

color:white;

border:1 solid white;

border-radius: 5px;

}</string>
        </property>
        <property name="text">
         <string>suuprimer</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>130</y>
          <width>141</width>
          <height>131</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {

gridline-color: white;

color:white;

background-color:rgba(0,0,0,50%);

border:0px;

border-radius: 5px;

}</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>rendezvous</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>591</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{

background:#EA9D9D;

color:white;

}

QTabBar::tab:selected{

background:#76A4F4;

color:white;

}

QTabWidget::pane { /* The tab widget frame */

border-top: 0px solid #EA9D9D;

}

QTabWidget::tab-bar {

left: 5px; /* move to the right by 5px */

}

tab</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>110</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>160</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>donnez le numéro</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>dinnez la date</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>300</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{

background-color:rgba(0,0,0,50%);

color:white;

border:1 solid white;

border-radius: 5px;

}</string>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>311</width>
          <height>321</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {

gridline-color: white;

color:white;

background-color:rgba(0,0,0,50%);

border:0px;

border-radius: 5px;

}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>250</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{

background-color:rgba(0,0,0,50%);

color:white;

border:1 solid white;

border-radius: 5px;

}</string>
        </property>
        <property name="text">
         <string>pdf</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{

background-color:rgba(0,0,0,50%);

color:white;

border:1 solid white;

border-radius: 5px;

}</string>
        </property>
        <property name="text">
         <string>imprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>donnez le numéro</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>180</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>saisir la nouvelle date</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>180</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>260</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{

background-color:rgba(0,0,0,50%);

color:white;

border:1 solid white;

border-radius: 5px;

}</string>
        </property>
        <property name="text">
         <string>modiifer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>supprimer</string>
       </attribute>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>donnez le numéro</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_10">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>110</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>230</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{

background-color:rgba(0,0,0,50%);

color:white;

border:1 solid white;

border-radius: 5px;

}</string>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>160</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {

gridline-color: white;

color:white;

background-color:rgba(0,0,0,50%);

border:0px;

border-radius: 5px;

}</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuactrendez">
    <property name="title">
     <string>actrendez</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuactrendez"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
